cpp/cmake_modules/FindArrowDataset.cmake

```diff
@@ -44,9 +44,8 @@ if(${CMAKE_FIND_PACKAGE_NAME}_FIND_QUIETLY)
   list(APPEND find_package_arguments QUIET)
 endif()
 find_package(Arrow ${find_package_arguments})
-find_package(Parquet ${find_package_arguments})
 
-if(ARROW_FOUND AND PARQUET_FOUND)
+if(ARROW_FOUND)
```

Comment:
@kou Do you have any concerns about the CMake changes here?

Reply:
No problem.